assets/app/views/modals/process-template.html

```diff
@@ -0,0 +1,21 @@
+<div class="modal-resource-action">
+  <div class="modal-body">
+    <h1>You are adding a Template</h1>
+    <p>Please select type of actions you want to do with your defined Template:</p>
```

Comment:
@jwforres had some suggested wording for this dialog. Have you updated the text?